packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.ts

```diff
@@ -23,7 +23,10 @@ export class NodeHttpRequestResponseHandler implements NodeHttpStreamsHandler {
    *
    * @param { HttpHandler } httpHandler - the handler through which to pass incoming requests.
    */
-  constructor(private httpHandler: HttpHandler) {
+  constructor(
+    private httpHandler: HttpHandler,
+    private hsts: { maxAge: number; includeSubDomains: boolean } = { maxAge: 7200, includeSubDomains: true },
```

Comment:
a server that handles only http requests should not be responding with `strict-transport-security` headers

Reply:
@woutermont could you take a look at this? The guys are waiting for your response :-)

Reply:
Done. Decided with @BelgianNoise to make the parameter optional.